### ISSUE TITLE
Bug in demo - reports missing indicators.csv

### ISSUE THREAD

**Issue by andre-geldenhuis** [voila-dashboards/voici#92]
## Description

Running the demo, it fails in the web interface saying it can't find indicators.csv

## Reproduce

In a fresh python venv, install voici, jupyterlite-xeus-python.  Clean up build space of micoconda, just in case. Then build the demo project as is from the cloned repo.

```bash
rm /tmp/xeus-python-kernel. # just in case
python3 -m venv env
source env/bin/activate

pip install voici jupyterlite-xeus-python
cd demo
voici build --contents notebooks
cd _output
python -m http.server

#fails at loading the csv in the browser
```
 

## Expected behavior

Succeeds at loading the dataframe

## Context

python 3.10
MacOS
Firefox and Chrome both do it.

The widgets directory notebooks do run, both bqplot.ipynb and ipycanvas.ipynb

pip freeze output in case it's useful
```
annotated-types==0.5.0
anyio==4.0.0
appdirs==1.4.4
argon2-cffi==23.1.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
attrs==23.1.0
Babel==2.12.1
beautifulsoup4==4.12.2
bleach==6.0.0
certifi==2023.7.22
cffi==1.15.1
charset-normalizer==3.2.0
click==8.1.7
cloudpickle==2.2.1
defusedxml==0.7.1
doit==0.36.0
empack==3.1.4
fastjsonschema==2.18.0
fqdn==1.5.1
idna==3.4
importlib-metadata==6.8.0
isoduration==20.11.0
Jinja2==3.1.2
json5==0.9.14
jsonpointer==2.4
jsonschema==4.19.1
jsonschema-specifications==2023.7.1
jupyter-events==0.7.0
jupyter_client==8.3.1
jupyter_core==5.3.1
jupyter_server==2.7.3
jupyter_server_terminals==0.4.4
jupyterlab-pygments==0.2.2
jupyterlab_server==2.25.0
jupyterlite-core==0.1.2
jupyterlite-xeus-python==0.9.4
MarkupSafe==2.1.3
mistune==3.0.1
nbclient==0.7.4
nbconvert==7.8.0
nbformat==5.9.2
networkx==3.1
overrides==7.4.0
packaging==23.1
pandocfilters==1.5.0
platformdirs==3.10.0
prometheus-client==0.17.1
ptyprocess==0.7.0
pycparser==2.21
pydantic==2.4.0
pydantic_core==2.10.0
Pygments==2.16.1
python-dateutil==2.8.2
python-json-logger==2.0.7
PyYAML==6.0.1
pyzmq==25.1.1
referencing==0.30.2
requests==2.31.0
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rpds-py==0.10.3
Send2Trash==1.8.2
six==1.16.0
sniffio==1.3.0
soupsieve==2.5
terminado==0.17.1
tinycss2==1.2.1
tornado==6.3.3
traitlets==5.10.0
typer==0.9.0
typing_extensions==4.8.0
uri-template==1.3.0
urllib3==2.0.5
voici==0.4.6
voila==0.5.0a4
webcolors==1.13
webencodings==0.5.1
websocket-client==1.6.3
websockets==11.0.3
zipp==3.17.0
```

**Comment by andre-geldenhuis** [comment]
A github actions deploy does work though.  So it is something on my end for sure.

**Comment by jtpio** [comment]
The demo on this repo seems to be working fine: https://voici.readthedocs.io/en/latest/_static/lite/voici/render/voici.html

![image](https://github.com/voila-dashboards/voici/assets/591645/50c0e661-d4ab-4b24-8a3c-4bdf4f1f5603)

@andre-geldenhuis which browser are you using? There are some know issues for example when running in Firefox in a private window: https://github.com/jupyterlite/jupyterlite/issues/679

**Comment by jtpio** [comment]
In a Firefox private window there is indeed the error:

![image](https://github.com/voila-dashboards/voici/assets/591645/34fe67b4-45bb-438c-b26e-260dcbcfad08)

**Comment by andre-geldenhuis** [comment]
Thanks for getting back so fast!  

It seems to happen in both firefox and Chrome, with or without private windows.  It does work fine from the github action build though. Even in the same browsers.

It might be a MacOS thing?  I'll try on a linux box

**Comment by trungleduc** [comment]
Hi, can you try with Voici 0.5.0?

**Comment by pjaggi1** [comment]
> In a Firefox private window there is indeed the error:
> 
> ![image](https://user-images.githubusercontent.com/591645/270556059-34fe67b4-45bb-438c-b26e-260dcbcfad08.png)
Experienced this with firefox private window.  however, switching to a public firefox window and clearing its cache resolved the issue.

**Comment by trungleduc** [comment]
This should be fixed in 0.6.0. Please feel free to re-open if needed.
